### PR TITLE
167 add network cascading interdependency analysis notebook

### PR DESCRIPTION
**Pull request by nunezco2**
This PR adds network cascading interdependency functionality to incore-docs.

### CODE REVIEW

File: notebooks/nci_functionality.ipynb [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "result = nic_func..get_output_dataset(\"epfDiscretizedCascadingFunc\")\n",
```
Comment (navarroc) [inline notebooks/nci_functionality.ipynb:313]: Typo, two .. and this should be:

result = nic_func.get_output_dataset("epf_cascading_functionality")

Comment (nunezco2) [reply notebooks/nci_functionality.ipynb:313]: Fixed

**Comment by navarroc** [comment]
Looks good, can you fix the merge conflict?

**Comment by nunezco2** [comment]
Done

**Review by navarroc — APPROVED** [review]

**Comment by longshuicy** [comment]
image
i saw this warning... is it important?

**Comment by longshuicy** [comment]
Could you resolve the conflict in changelog?

**Review by longshuicy — APPROVED** [review]

**Pull request merged**